index.html

```diff
@@ -777,7 +781,8 @@ <h4>
         <a data-cite="webdriver2/#dfn-current-browsing-context">current browsing
         context</a>'s [=top-level traversable=]'s [=navigable/active document=].
         </li>
-        <li>Set |document|.{{Document/[[PostureOverride]]}} to
+        <li>Set |document|.<a data-link-for="top-level traversable">
```

Comment:
This is still wrong.

Reply:
Could you verbose a little bit. Which part is wrong? Code or the HTML it produces?
|document| in here doesn't mean the "Document" which every e.g. iframe has, but is variable of current browsing context's top-level traversable's active document. So should not be the `|document|` part. There should not be `.`  between `|document|` and `<a data-link-for` ?

Reply:
We've been talking specifically about this for days so I thought I didn't need to elaborate further. If you say that `[[PostureOverride]]` is an internal slot of the top-level traversable, then accessing it from a Document is wrong.

Reply:
So the problems starts from already earlier step?

**Text**
"
Let _document_ be the current browsing context's top-level traversable's active document.
Set _document_.[[PostureOverride]] to _null_.
"
==>
"
Let _topLevelTraversable_ be the current browsing context's top-level traversable.
Set _topLevelTraversable_.[[PostureOverride]] to _null_.
"

**Code**
```
<li>Let |document| be the
<a data-cite="webdriver2/#dfn-current-browsing-context">current browsing
context</a>'s [=top-level traversable=]'s [=navigable/active document=].
</li>
<li>Set |document|.<a data-link-for="top-level traversable">
[[\PostureOverride]]</a> to <code>null</code>.
```
==>
```
<li>Let |topLevelTraversable| be the
<a data-cite="webdriver2/#dfn-current-browsing-context">current browsing
context</a>'s [=top-level traversable=].
</li>
<li>Set |topLevelTraversable|.<a data-link-for="top-level traversable">
[[\PostureOverride]]</a> to <code>null</code>.
```

Reply:
> So the problems starts from already earlier step?

Yes.

Your proposed fix is almost correct, `[=top-level traversable=]` will refer to the concept, not the browsing context's top-level traversable, which you get with `[=browsing context/top-level traversable=]`. Again, this is all in the Generic Sensor spec.

Reply:
https://github.com/JuhaVainio/W3C-device-posture/pull/1/commits/9192ea0141132ca88512739cf0d0d7db8e39dd40

Damn there was still that.